README.md

```diff
@@ -56,7 +57,7 @@ Table of Contents
 
 ## Quick Start
 
-The descheduler can be run as a Job or CronJob inside of a k8s cluster. It has the
+The descheduler can be run as a Job or CronJob or Deployment inside of a k8s cluster. It has the
```

Comment:
The descheduler can be run as a Job, CronJob, or Deployment inside of a k8s cluster.

Reply:
Fixing this!